### DISCUSSION TITLE
SSE Support for Subscriptions

### DISCUSSION THREAD

**Discussion by johannbrynjar** [MichalLytek/type-graphql#1456] [Q&A]
I have been exploring real-time data updates and came across Server-Sent Events (SSE) as a lightweight and efficient option for server-to-client communication. 

I wonder if it would be possible to integrate SSE support for subscriptions in Type-GraphQL?

Best regards,
Jóhann

**Comment by MichalLytek** [comment]
https://the-guild.dev/graphql/yoga-server/docs/features/subscriptions#graphql-over-server-sent-events-protocol-via-graphql-sse

**Reply by AChangXD** [reply]
Looking into SSE as well. If I'm already using apollo server, I'd just have to spin up an instance of yogaServer on another port?
